.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,298 @@
+---
+name: agw-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  lte-test:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install libraries and dependencies
+        env:
+          PYTHON_BUILD: "${{ env.MAGMA_ROOT }}/build"
+          PIP_CACHE_HOME: "${{ env.MAGMA_ROOT }}/.pipcache"
+          MAGMA_DEV_MODE: 1
+          SKIP_SUDO_TESTS: 1
+          CODEGEN_ROOT: "${{ env.MAGMA_ROOT }}/.codegen"
+          SWAGGER_CODEGEN_JAR: "${{ env.MAGMA_ROOT }}/.codegen/swagger-codegen-cli.jar"
+        run: |
+            mkdir -p /var/tmp/test_results
```

Comment:
missing https://github.com/magma/magma/blob/206c84171a3fdd477cb6290750cd967138c92c1f/.circleci/config.yml#L212

Reply:
You are right!.. added